src/common/pf_snmp.c

```diff
@@ -184,9 +184,7 @@ void pf_snmp_get_system_description (
 
 void pf_snmp_get_port_list (pnet_t * net, pf_lldp_port_list_t * p_list)
 {
-   /* TODO: Implement support for multiple ports */
-   memset (p_list, 0, sizeof (*p_list));
-   p_list->ports[0] = BIT (8 - 1);
+   pf_lldp_get_port_list (net, p_list);
 }
 
 int pf_snmp_get_first_port (const pf_lldp_port_list_t * p_list)
```

Comment:
Maybe
```
int pf_snmp_get_first_port (const pf_lldp_port_list_t * p_list)
{
   return pf_lldp_get_first_port (p_list);
}

int pf_snmp_get_next_port (const pf_lldp_port_list_t * p_list, int loc_port_num)
{
   return pf_lldp_get_next_port (p_list, loc_port_num);
}
```

And 

```
int pf_lldp_get_first_port (const pf_lldp_port_list_t * p_list)
{
   /* TODO: Implement support for multiple ports */
   return 1;
}

int pf_lldp_get_next_port (const pf_lldp_port_list_t * p_list, int loc_port_num)
{
   /* TODO: Implement support for multiple ports */
   return 0;
}
```

And corresponding in `pf_lldp.h`.Then it will be obvious what details that needs to be implemented in the lldp files.

Reply:
I'm not sure these functions actually belongs in the pf_lldp.c. The list of ports doesn't seem to be sent in LLDP frames, unless I've missed it.

Reply:
Ok, seems reasonable.

Reply:
Maybe you can just put a comment ("not part of LLDP frames" or something similar) in the TODO to help me and others to understant where it should be implemented.

Reply:
Good suggestion. Will do.

Reply:
Moved to pf_lldp_c for now.